x/session/types/query_get_session_request.go

```diff
@@ -27,8 +26,8 @@ func (query *QueryGetSessionRequest) ValidateBasic() error {
 	}
 
 	// Validate the Service ID
-	if !sharedhelpers.IsValidService(query.Service) {
-		return ErrSessionInvalidService.Wrapf("invalid service for session being retrieved %s;", query.Service)
+	if err := query.Service.ValidateBasic(); err != nil {
+		return ErrSessionInvalidService.Wrapf("invalid service for session being retrieved %s; %v", query.Service, err)
```

Comment:
What was the rationale behind adding the `%v` here?

```suggestion
		return ErrSessionInvalidService.Wrapf("%s", err)
```

Reply:
Probably copied from somewhere else without much thoughts. Will fix.